code/game/objects/structures/extinguisher.dm

```diff
@@ -48,6 +51,27 @@
 		opened = !opened
 	update_icon()
 
+/obj/structure/extinguisher_cabinet/attackby(obj/item/W as obj, mob/user as mob, params)
+	if(opened)
+		if(istype(W, /obj/item/weldingtool))
+			var/obj/item/weldingtool/WT = W
+			if(!WT.remove_fuel(0, user))
+				to_chat(user, "<span class='notice'>You need more welding fuel to complete this task.</span>")
+				return
+			to_chat(user, "<span class='notice'>You begin cutting \the [src] apart...</span>")
+			playsound(loc, cutting_sound, 40, 1)
+			if(do_after(user, 40, 1, target = src))
+				if(!opened || !WT.isOn())
+					return
+				playsound(loc, cutting_sound, 50, 1)
```

Comment:
Ditto

Reply:
As above, you want `playsound(loc, W.usesound, 50, 1)` to utilize the sound of the tool being used.

Also you seem to have a difference of 10 on the volumes between starting and completing. Not major, but pointing it out.